personal_test.go

```diff
@@ -40,23 +43,80 @@ func TestPersonal_User(t *testing.T) {
 
 //}
 
-//func TestPersonal_Transactions(t *testing.T) {
-//	transactions := []Transaction{
-//		{
-//			ID:              "ZuHWzqkKGVo=",
-//			Time:            1554466347,
-//			Description:     "Покупка щастя",
-//			MCC:             7997,
-//			Hold:            false,
-//			Amount:          -95000,
-//			OperationAmount: -95000,
-//			CurrencyCode:    980,
-//			CommissionRate:  0,
-//			CashBackAmount:  19000,
-//		},
-//	}
-//
-//}
+func TestPersonal_Transactions(t *testing.T) {
```

Comment:
Unfortunately, we shouldn't put this changes into the PR. It is out of the scope. 
Our tests are pretty poor, let's keep these changes for PR for #29.